[IMP] accounting: update the follow-up information (#1834)
odoo/documentation: content/applications/finance/accounting/receivables/customer_payments/followup.rst

=====================
Follow-up on invoices
=====================

A follow-up message can be sent to customers when a payment is overdue. Odoo helps you identify late
payments and allows you to schedule and send the appropriate reminders. Your follow-ups can be sent
via different means such as email, post, or even by SMS.

.. seealso::
   Odoo tutorials: Payment follow-up (LINK TO NEW VIDEO TO BE ADDED)

Follow-up Levels configuration
==============================

A follow-up plan, with several actions, is available by default in :menuselection:`Accounting -->
Configuration --> Follow-up Levels`, but you can customize it any way you want.

Depending on the number of overdue days you decide, you can plan to print a letter, send an email,
an SMS message, or a letter, to follow up on with your customer. It is also possible to

> automatically send a reminder by checking the *Auto Execute* option

Author: Sentence updated as follow: Depending on the number of overdue days you set in the *Follow-up Action* form, you can plan to print a letter, send an email, an SMS message, or a letter to follow up with your customer. It is
also possible to automatically send a reminder by enabling the *Auto Execute* option.

.. tip::
   To send a reminder before the actual due date is reached, set a negative number of due days.

> Follow-up Reports

Author: Le terme est en majuscule dans Odoo... 
![image](https://user-images.githubusercontent.com/85342080/168236287-f53be2ac-3b00-4ffb-bc64-ea643abc8ff8.png)

> Follow-up Reports

Author: du coup, majuscule ou minuscule? :)

Follow-up Reports
=================

.. image:: media/followup01.png
    :align: center
The overdue invoices you need to follow up on are available in :menuselection:`Accounting -->
Customers --> Follow-up Reports`. From there, you  can identify all the customers that have late
unpaid invoices.

Send reminders in batch
=======================
When you open a customer, you see all their unpaid invoices (overdue or not). The due dates of the
late invoices appear in red. Select the invoices that are not late yet in the *Excluded* column to
exclude them from the reminder you send.

To get your follow-up process easier, you can send reminder emails in
batch from your follow-up reports page. Select all the reports you would
like to process, click on *Action* and then on *Process Follow-ups*.
A pdf document containing all of the follow-up letters for you to print
will also be automatically generated.
It is up to you to decide how to remind your customer. You can select *Print Letter*,
*Send By Email*, *Send By Post*, *Send By SMS*. Then, click on the *Done* button to view the next
follow-up that needs your attention.

.. image:: media/followup02.png

>     :align: center

Author: I no longer see it...

>    * The contact information available on the invoice or on the contact is used to send the

Author: The contact information available on the invoice or on the contact form is used for? 
"used to send the reminder." isn't correct?

> them as good, normal, or bad debtors on the follow-up reports. Just click on the button next to the

Reviewer: ```suggestion
them as good, normal, or bad debtor on the follow-up report. Just click on the bullet next to the
```

> Send reminders in batch

Reviewer: ```suggestion
Send reminders in batches
```

> To make your follow-up process more manageable, you can send reminder emails in batches from your

Reviewer: ```suggestion
To make your follow-up process even smoother, you can send reminder emails in batches from the
```

> Follow-up Reports page. Select all the reports you would like to process, click on Action, and then

Reviewer: ```suggestion
Follow-up Reports page. Select all the reports you would like to process, click on the *Action* gear icon, and select
```

> on Process Follow-ups.

Reviewer: ```suggestion
*Process Follow-ups*.
```
